### PR TITLE
#146683083 Convert from Grunt to Gulp taskrunner

### PR DESCRIPTION
**Pull request by Tawakalt**
#### What does this PR do?
- It converts from Grunt to Gulp taskrunner

#### Description of Task to be completed?

Create gulpfile.js and install dependencies
Delete gruntfile.js and remove dependencies 
Ignore bower_components
Modify scripts to use gulp in package.json.

#### How should this be manually tested?
Running npm start and npm test OR gulp and gulp test

#### Any background context you want to provide?
NONE

#### What are the relevant pivotal tracker stories?
NONE

#### Screenshots (if appropriate)

screen shot 2017-06-13 at 1 46 36 pm
screen shot 2017-06-13 at 1 48 46 pm
screen shot 2017-06-13 at 1 49 49 pm
screen shot 2017-06-13 at 1 53 02 pm
screen shot 2017-06-13 at 1 53 18 pm

#### Questions:
NONE

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+const sass = require('gulp-sass');
+
+gulp.task('watch', () => {
+  gulp.watch('app/views/**', browserSync.reload());
+  gulp.watch('public/js/**', browserSync.reload());
+  gulp.watch('app/**/*js', browserSync.reload());
```
Comment (tonerdo) [inline gulpfile.js]: Shouldn't this be `app/**/*.js`?

**Review by tonerdo — CHANGES_REQUESTED** [review]
A quick one and we should be good

**Review by tonerdo — CHANGES_REQUESTED** [review]
A quick one and we should be good

File: gulpfile.js (hunk 1 above)
Comment (Tawakalt) [reply gulpfile.js]: Yes. Fixed it

**Comment by tonerdo** [comment]
@Tawakalt kindly fix the merge conflicts and we'll be good

**Review by tonerdo — APPROVED** [review]

**Pull request merged**